src/category_theory/limits/creates.lean

```diff
@@ -122,6 +122,19 @@ lemma has_limit_of_created (K : J ⥤ C) (F : C ⥤ D)
   [has_limit (K ⋙ F)] [creates_limit K F] : has_limit K :=
 has_limit.mk { cone := lift_limit (limit.is_limit (K ⋙ F)),
   is_limit := lifted_limit_is_limit _ }
+  
+/-- 
+If `F` creates limits of shape `J`, and `D` has limits of shape `J`, then 
+`C` has limits of shape `J`. 
+-/ 
+def has_limits_of_shape_of_has_limits_of_shape_creates_limits_of_shape (F : C ⥤ D)
+  [has_limits_of_shape J D] [creates_limits_of_shape J F] : has_limits_of_shape J C := 
+⟨λ G, has_limit_of_created G F⟩
+  
+/-- If `F` creates limits, and `D` has all limits, then `C` has all limits. -/
+def has_limits_of_has_limits_creates_limits (F : C ⥤ D) [has_limits D] [creates_limits F] :
+  has_limits C := 
+⟨λ J I, by exactI (has_limits_of_shape_of_has_limits_of_shape_creates_limits_of_shape F)⟩
```

Comment:
You could even drop the parentheses here:
```suggestion
⟨λ J I, by exactI has_limits_of_shape_of_has_limits_of_shape_creates_limits_of_shape F⟩
```

Reply:
Done!